css/style.css

```diff
@@ -16,7 +16,9 @@
 
 .sub-container h2::after,
 .partners h3::after,
-.featured h2::after {
```

Comment:
- [ ] As requested by the previous revouwer here 👇🏻 , Kindly make sure to address the requested changes :wink: 💯 

- [ ] In the 'Main program' Section, kindly add gaps between the cards to separate them.
    - [ ] Kindly put this section in a section tag and not a div, for each article place then in article tags and not divs.
   > ![Screenshot 2022-05-20 015455](https://user-images.githubusercontent.com/77142475/169422905-005f822d-581b-4e41-adfc-614b9fea0878.png)